### DISCUSSION TITLE
After update to 9.0.1 database container won't start

### DISCUSSION THREAD

**Discussion by bbukowski** [nextcloud/all-in-one#4892] [Questions] [answered]
Hello together,

after updating to 9.0.1 the database container won't start:

> The database import failed. Please restore a backup and try again.
> For further clues on what went wrong, look at the logs above.

The _database-import.log_ ends with:

> + DIFFERENT_DB_OWNER=1
> + psql -v ON_ERROR_STOP=1 --username nextcloud --dbname nextcloud_database
> /start.sh: line 110: cannot create temp file for here-document: Read-only file system

Please see the whole log in the attachments.
[database-import.log](https://github.com/user-attachments/files/15978978/database-import.log)

I am a bit lost where look right now. So any help is appreciated.

Thanks in advance,
Benjamin

**Comment by szaimen** [comment]
Hi, which docker version are you running? 

Also can you restore a backup and check if that makes it work?

**Comment by bbukowski** [comment]
I already tried restoring a backup, which ends up the same.

> Docker version 27.0.1, build 7fafd33

**Comment by szaimen** [comment]
> * /start.sh: line 110: cannot create temp file for here-document: Read-only file system

Is there maybe something more verbose in the logs? Where is the location where it tries to create the temp file for this `here-document` ?

**Reply by bbukowski** [reply]
[This](https://github.com/nextcloud/all-in-one/blob/64bdfb535cd511c8ba334d911fdf392fe562db3f/Containers/postgresql/start.sh#L110) must be the line which fails for some reason.

In the logs I don't see anything which may related..

**Reply by szaimen** [reply]
Can you post the complete database logs here?

**Reply by bbukowski** [reply]
Sure, but which do you mean and how to fetch them?

**Reply by szaimen** [reply]
`sudo docker logs nextcloud-aio-database`

**Reply by bbukowski** [reply]
[nextcloud-aio-database.log](https://github.com/user-attachments/files/15984575/nextcloud-aio-database.log)

**Reply by szaimen** [reply, marked as answer]
![image](https://github.com/nextcloud/all-in-one/assets/42591237/aabe3333-d5a2-4e8b-9411-2f5b05a4f31d)
Based on that you ran into a problem that is fixed with v9.1.0 Beta. Can you [switch to beta](https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-switch-the-channel) and restore a backup and try if that fixes it?

**Reply by bbukowski** [reply]
Thank you very much, that fixed it!
